air-interpreter/Cargo.toml

```diff
@@ -26,4 +26,4 @@ serde = { version = "=1.0.118", features = [ "derive", "rc" ] }
 serde_json = "1.0"
```

Comment:
1.0.56 ?

Reply:
We could fix it, yep, but is it worth so?

Reply:
probably not

Reply:
It uses 1.0.61 judging by Cargo.lock